packages/@react-stately/virtualizer/src/Virtualizer.ts

```diff
@@ -867,7 +867,8 @@ export class Virtualizer<T extends object, V, W> {
 
       let layoutInfo = this.layout.getLayoutInfo(cur.key);
       if (
-        !cur.rect.pointEquals(layoutInfo.rect) ||
+        // Uses equals rather than pointEquals so that width/height changes are taken into account
+        !cur.rect.equals(layoutInfo.rect) ||
```

Comment:
If all columns were removed/hidden except the left most column, the `x, y` position of the rect doesn't shift but the width changes. Changed to `rect.equals` instead of `rect.pointEquals` so that this case is properly captured